awx/main/models/credential/injectors.py

```diff
@@ -18,8 +18,6 @@ def gce(cred, env, private_data_dir):
     project = cred.get_input('project', default='')
     username = cred.get_input('username', default='')
 
-    env['GCE_EMAIL'] = username
-    env['GCE_PROJECT'] = project
```

Comment:
See https://docs.ansible.com/ansible/latest/modules/gce_module.html

I can find no evidence that these are employed by the gce _module_ anywhere, and if not, they shouldn't be here.

Reply:
https://github.com/ansible/ansible/blob/276ad32a4535c262b9aa42c08981656344959f53/lib/ansible/module_utils/gcp.py#L110-L111

>        b) GCE_PROJECT, GCE_CREDENTIAL_FILE_PATH, GCE_EMAIL (legacy, not recommended; req'd if using p12 key)

Maybe I'm wrong about this and GCE_EMAIL is still needed...